client/store/assessment/actions/initApp.ts

```diff
@@ -0,0 +1,15 @@
+import { createAsyncThunk } from '@reduxjs/toolkit'
+
+import axios from 'axios'
+import { ApiEndPoint } from '@common/api/endpoint'
+
+export const initApp = createAsyncThunk('app/init', async () => {
+  try {
+    return axios.get(ApiEndPoint.Init.one())
+  } catch (err) {
+    // if (err.response && err.response.status !== 401) {
+    //   dispatch(applicationError(err))
+    // }
```

Comment:
Commented off until error handling added

Reply:
we dont need the catch here because of the middleware?